std/process.d

```diff
@@ -1443,6 +1499,33 @@ int wait(Pid pid) @safe
     else version (Posix) assert(pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "do { timeout 1 } while (1)";
```

Comment:
Well, it's not being executed as a PowerShell script. Look at the implementation of `TestScript`. The string is saved to a `.cmd` file. In addition to that, I don't think you can just `CreateProcess` / `spawnProcess` a `.cmd` etc. file on Windows. Windows does not have shebangs, associations (incl. `.cmd`) are handled by userspace (the shell) in a layer above `CreateProcess`, so you must spawn the command interpreter explicitly.